src/index.js

```diff
@@ -1,12 +1,159 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
-import App from './App';
-import * as serviceWorker from './serviceWorker';
 
-ReactDOM.render(<App />, document.getElementById('root'));
+function Square(props) {
 
-// If you want your app to work offline and load faster, you can change
-// unregister() to register() below. Note this comes with some pitfalls.
-// Learn more about service workers: https://bit.ly/CRA-PWA
-serviceWorker.unregister();
+    return (
+        <button className="square" onClick={props.onClick}>
+            {props.value}
+        </button>
+    );
+}
+
+
+class Board extends React.Component {
+
+    renderSquare(i) {
+        return (
+         <Square
+            value={this.props.squares[i]}
+            onClick={() => this.props.onClick(i)}
+        />
+        );
+    }
+
+    render() {
+        return (
+           <div>
+             <div className="board-row">
+                {this.renderSquare(0)}
```

Comment:
```suggestion
                {Array(9).map((el, i) => this.renderSquare(i))}
```
It can help you to get rid of manual rendering